source/features/mark-merge-commits-in-list.tsx

```diff
@@ -0,0 +1,63 @@
+import './mark-merge-commits-in-list.css';
+import select from 'select-dom';
+import gitmerge from 'octicon/git-merge.svg';
+import * as api from '../libs/api';
+import features from '../libs/features';
+import {getRepoGQL} from '../libs/utils';
+
+const getMergeCommits = async (commits: string[]): Promise<string[]> => {
+	const {repository} = await api.v4(`
+		repository(${getRepoGQL()}) {
+			${commits.map((commit: string) => `
+				${api.escapeKey(commit)}: object(expression: "${commit}") {
+					... on Commit {
+							parents{
+								totalCount
+							}
+						}
+					}
+			`).join('\n')}
+		}
+	`);
+
+	return filterCommits(repository);
+};
+
+function filterCommits(commits: object): string[] {
+	const mergeCommits = [];
+	for (const commit of Object.entries(commits)) {
```

Comment:
Destructure, so you don’t have to use `commit[1]` later 
```suggestion
	for (const [key, commit] of Object.entries(commits)) {
```

Reply:
Please follow my reviews fully, especially when I provide exact code suggestions. I used `key` because the value will be the API's escaped key, not the commit. It becomes `commit` only after you apply `.slice(1)`

Reply:
No problem.

Just go easy on me. I am working hard and I am also fighting with typescript too.

Reply:
Is this the only issue?

Reply:
@fregante 
the return value is 
```javascript
{_0341d299c80dbfc8f4aabbb4d36829db8cf7f2a9:
parents:
totalCount: 1}
```
so key would be the commit and the second value is the parents. Can You give me a drop more detail where i'm going wrong?

Reply:
Copy-paste my code suggestion, that’s all that went wrong